scanner_cli.py

```diff
@@ -5,15 +5,18 @@
 """
 
 import argparse
+import json
 import csv
 from pip._vendor import requests
 
+API_KEY = ""
```

Comment:
API_KEY can be an enviornment variable - `os.getenv("API_KEY")` and just do `export API_KEY=xxxx` in your terminal shell

Reply:
Yeah, I can change this. Was like this because initially you said to just have the key hardcoded in for now